baanders2021reproducibility/openreview/packages.tex

```diff
@@ -0,0 +1,18 @@
+\usepackage[final]{neurips_2019}
```

Comment:
Please remove this import - this is why `make` fails but compiling in OpenReview folder works, as it tries to load an incorrect style file (we use the rescience.cls file)

Reply:
Did this and pushed. Hope it works now, Thanks for the feedback